examples/computer_vision/detectron2_coco_pytorch/model_def.py

```diff
@@ -73,8 +74,8 @@ def build_validation_data_loader(self):
         return data_loader
 
     def train_batch(self, batch: TorchData, epoch_idx: int, batch_idx: int):
-
-        loss_dict = self.model(batch)
+        with EventStorage() as storage:
```

Comment:
if you are not actually using `storage` you can omit `as storage` and just say `with EventStorage():`